apps/passport-server/test/genericIssuance.spec.ts

```diff
@@ -107,56 +111,67 @@ describe("Generic Issuance", function () {
+   * Attendee ticket type. In reality there will be several.
    *
-   * TODO: test that we can handle several attendee tiers.
+   * TODO: test that we can handle several attendee types.
    */
-  const EdgeCityAttendeeTier: LemonadeTicketTier = lemonadeBackend.addTier(
-    EdgeCityDenver.id,
-    "ga"
-  );
-  const EdgeCityBouncerTier = lemonadeBackend.addTier(
-    EdgeCityDenver.id,
+  const EdgeCityAttendeeTicketType: LemonadeTicketType =
+    EdgeCityLemonadeAccount.addTicketType(EdgeCityDenver._id, "ga");
+  const EdgeCityBouncerTicketType = EdgeCityLemonadeAccount.addTicketType(
+    EdgeCityDenver._id,
     "bouncer"
   );
 
   /**
    * Most tests below need a person who is checking tickets {@link EdgeCityDenverBouncer}
    * and a person whose ticket needs to be checked in (@link Attendee)
    */
-  const EdgeCityDenverAttendee: LemonadeUser =
-    lemonadeBackend.addUser("attendee");
-  const EdgeCityDenverAttendeeIdentity = new Identity();
-  const EdgeCityAttendeeTicket: LemonadeTicket = lemonadeBackend.addTicket(
-    EdgeCityAttendeeTier.id,
-    EdgeCityDenver.id,
-    EdgeCityDenverAttendee.name,
-    EdgeCityDenverAttendee.email
+  const EdgeCityDenverAttendee: LemonadeUser = lemonadeBackend.addUser(
+    "attendee@example.com",
+    "attendee",
+    "smith"
   );
+  const EdgeCityDenverAttendeeIdentity = new Identity();
+  const EdgeCityAttendeeTicket: LemonadeTicket =
+    EdgeCityLemonadeAccount.addTicket(
+      EdgeCityDenver._id,
+      EdgeCityAttendeeTicketType._id,
+      EdgeCityDenverAttendee._id,
+      `${EdgeCityDenverAttendee.first_name} ${EdgeCityDenverAttendee.last_name}`
+    );
 
   /**
    * Similar to {@link EdgeCityDenverAttendee}
    * Person who has a {@link LemonadeTicket} that does not have a bouncer ticket,
    * i.e. a ticket whose 'product id' or 'tier' is set up to be a 'superuser' ticket
    * by the Generic Issuance User with id {@link edgeCityGIUserID}.
    */
-  const EdgeCityDenverBouncer: LemonadeUser =
-    lemonadeBackend.addUser("bouncer bob");
+  const EdgeCityDenverBouncer: LemonadeUser = lemonadeBackend.addUser(
+    "bouncer@example.com",
+    "bouncer",
+    "bob"
+  );
   const EdgeCityBouncerIdentity = new Identity();
-  const EdgeCityDenverBouncerTicket = lemonadeBackend.addTicket(
-    EdgeCityBouncerTier.id,
-    EdgeCityDenver.id,
-    EdgeCityDenverBouncer.name,
-    EdgeCityDenverBouncer.email
+  const EdgeCityDenverBouncerTicket = EdgeCityLemonadeAccount.addTicket(
+    EdgeCityDenver._id,
+    EdgeCityBouncerTicketType._id,
+    EdgeCityDenverBouncer._id,
+    `${EdgeCityDenverBouncer.first_name} ${EdgeCityDenverBouncer.last_name}`
   );
-  lemonadeBackend.makeCoHost(EdgeCityDenverBouncer.id, EdgeCityDenver.id);
-  const lemonadeAPI: ILemonadeAPI = new MockLemonadeAPI(lemonadeBackend);
+
+  const lemonadeAPI: ILemonadeAPI = new LemonadeAPI({
+    async getToken(credentials: LemonadeOAuthCredentials): Promise<string> {
+      return credentials.oauthClientId;
```

Comment:
this seems like an opinionated choice for testing that ought to be documented with a comment - if i'm understanding correctly, in testing, the mocked lemonade backend accepts the oauth client id in place of some other token representation